src/metabase/cmd/rotate_encryption_key.clj

```diff
@@ -0,0 +1,35 @@
+(ns metabase.cmd.rotate-encryption-key
+  (:require [cheshire.core :as json]
+            [clojure.java.jdbc :as jdbc]
+            [metabase.db :as mdb]
+            [metabase.db.connection :as mdb.conn]
+            [metabase.models :refer [Database Setting]]
+            [metabase.util.encryption :as encrypt]
+            [toucan.db :as db]))
+
+(defn rotate-encryption-key!
+  "Rotate the current configured db using the current MB_ENCRYPTION_SECRET_KEY env var and `to-key` argument."
+  [to-key]
+  (mdb/setup-db!)
```

Comment:
where should I put this call? cmd.clj doesn't have calls to this function, but it doesn't feel right here either.  Or, maybe it's not needed? I'll recheck